drizzle/seed.ts

```diff
@@ -0,0 +1,313 @@
+          location: chance.country({ full: true }),
+          bio: chance.sentence({ words: 10 }),
+          websiteUrl: chance.url(),
+        };
+      });
+
+    return users;
+  };
+
+  const userData = generateUserData();
+  const communityData = generateCommunityData(10);
+
+  const addUserData = async () => {
+    const tags = sampleTags.map((title) => ({ title }));
+
+    const tagResponse = await db
+      .insert(tag)
+      .values(tags)
+      .onConflictDoNothing()
+      .returning({ id: tag.id, title: tag.title });
+
+    const usersResponse = await db.insert(user).values(userData).returning();
+
+    for (let i = 0; i < usersResponse.length; i++) {
+      const posts = randomPosts(
+        chance.integer({
+          min: 1,
+          max: 5,
+        }),
+      ).map((post) => ({ ...post, userId: usersResponse[i].id }));
+
+      const postsResponse = await db
+        .insert(post)
+        .values(posts)
+        .onConflictDoNothing()
+        .returning();
+
+      for (let j = 0; j < postsResponse.length; j++) {
+        const randomTag = tagResponse[chance.integer({ min: 0, max: 9 })];
+        await db
+          .insert(post_tag)
+          .values({ postId: postsResponse[j].id, tagId: randomTag.id })
+          .onConflictDoNothing();
+      }
+    }
+
+    console.log(`Added ${usersResponse.length} users with posts`);
+  };
+
+  const addEventData = async () => {
+    const communityResponse = await db
+      .insert(community)
+      .values(communityData)
+      .returning();
+
+    const allUsers = await db.select().from(user);
+
+    for (let i = 0; i < communityResponse.length; i++) {
+      const eventData = generateEventData(
+        chance.integer({
+          min: 1,
+          max: 5,
+        }),
+      ).map((event) => ({
+        ...event,
+        communityId: communityResponse[i].id,
+      }));
+
+      const community = communityResponse[i];
+      const membershipData = allUsers.map((user) => ({
+        id: nanoid(8),
+        userId: user.id,
+        communityId: community.id,
+        isEventOrganiser: user.id === allUsers[0].id,
+      }));
+
+      await db.insert(membership).values(membershipData).returning();
+      const eventsResponse = await db
+        .insert(event)
+        .values(eventData)
+        .returning();
+
+      for (let j = 0; j < eventsResponse.length; j++) {
+        const eventData = eventsResponse[j];
+        for (let k = 0; k < allUsers.length; k++) {
+          const userData = allUsers[j];
+          const id = nanoid(8);
+          await db
+            .insert(r_s_v_p)
+            .values({ eventId: eventData.id, id, userId: userData.id })
+            .onConflictDoNothing();
+        }
+      }
+    }
+
+    console.log(`Added events and members`);
+  };
+
+  async function addSeedDataToDb() {
+    console.log(`Start seeding, please wait... `);
+
+    try {
+      await addUserData();
+      await addEventData();
```

Comment:
```suggestion
      await Promise.all([addUserData(),addEventData()])
```

Maybe a small optimisation seeing as we dont seem to care about the return or the error

Reply:
We need the users first to add them to the event data so it might create a race condition